src/engine/ruleset/decoders/windows/windows-event-decoder.yml

```diff
@@ -0,0 +1,42 @@
+name: decoder/windows-event-decoder/0
+
+sources: 
+  - decoder/json/0
+
+metadata:
+  description: Parent decoder for Windows events
+  module: Windows
+  compatibility: >
+    This decoder has been tested on Wazuh version 4.3
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/23
+  title: This Windows decoders decodes basic field from Windows events
+  references:
+    - https://learn.microsoft.com/en-us/windows/win32/events/windows-events
+
+check:
+  - ~json.Event: +ef_exists
+
+parse:
+  logpar:
+    - ~json.Event: <~windows/xml/windows>
+
+normalize:
+ - map:
+   - timestamp: $~windows.Event.System.TimeCreated.@SystemTime
```

Comment:
can this field be changed to `event.start` ? We're using that approach on other similar cases

Reply:
Not sure why, but windows events do not have `event.start`

Reply:
the field @timestamp is automatically set by filebeat with the ingestion timestamp. The timestamp of the event goes in event.start.